tests/core/test_logging_config.py

```diff
@@ -143,15 +143,6 @@ def settings_context(content, directory=None, name='LOGGING_CONFIG'):
             # Create the directory structure
             dir_path = os.path.join(settings_root, directory)
             pathlib.Path(dir_path).mkdir(parents=True, exist_ok=True)
-
-            # Add the __init__ for the directories
-            # This is required for Python 2.7
-            basedir = settings_root
-            for part in directory.split('/'):
-                open(os.path.join(basedir, '__init__.py'), 'w').close()
```

Comment:
Are we sure changing those packages into "namespace" packages is a good idea here? I think it is the result of miconception that in Python 3.3 we can simply remove `__init__.py`  everywehere, but it is not really true.  See my comment here: https://github.com/apache/airflow/pull/15317#issuecomment-817300262 

I think  we should simply remove the comment but leave the `__init__.py` adding

Reply:
Agreed after reading your comment. I can always learn new things by reading your comments!